testsuite/pytests/test_sonata/300_pointneurons/network/internal_node_types.csv

```diff
@@ -0,0 +1,6 @@
+node_type_id ei model_template model_type dynamics_params model_name
+104 i nest:iaf_psc_alpha point_process 473862421_point.json PV2
+100 e nest:iaf_psc_alpha point_process 472363762_point.json Scnn1a

```

Comment:
Are all these files duplicated between ``examples`` and ``pytests``? Can we use a symbolic link or some other method of de-duplication (cc @jessica-mitchell)?

Reply:
While I agree it would be best to avoid duplication, I don't think there is a simple solution for this. The SONATA files should reside in the examples dir in order to be accessible to users. I tried to create a symlink in the testsuite dir pointing to the files in the examples dir. This doesn't work as the files are not found when, for instance, running `installcheck` from a build dir. The files comprise about 2.6 MB, so I don't think duplication matters too much when it comes to file size.